Northstar.CustomServers/mod/scripts/vscripts/item_inventory/sv_item_inventory.gnut

```diff
@@ -69,8 +69,17 @@ int function PlayerInventory_CountBurnRef( entity player, string burnRef )
 void function PlayerInventory_TakeInventoryItem( entity player )
 {
 	entity preexistingWeapon = player.GetOffhandWeapon( OFFHAND_INVENTORY )
-	if ( IsValid( preexistingWeapon ) )
-		player.TakeWeaponNow( preexistingWeapon.GetWeaponClassName() )
+
+	if( !IsValid( preexistingWeapon ) )
```

Comment:
formatting here needs some work, couple newlines around it and it should be good though